File: 2-Tarea-Operaciones-Basicas/E.La-banda-robatamales.java
import java.util.Scanner;

public class Main {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);

    int T = scanner.nextInt();
    int B = scanner.nextInt();

    int resultado = (T + 1) / 2;
    resultado += (T - resultado) % (B - 1);

    System.out.println(resultado);
  }
}

File: 2-Tarea-Operaciones-Basicas/E.La-banda-robatamales.cpp
#include <iostream>
using namespace std;

int main() {
  int T, B; 
  cin >> T >> B;

  int resultado = (T + 1) / 2;
  resultado += (T - resultado) % (B - 1);

  cout << resultado << endl;
  
  return 0;
}